mysite/account/templates/account/edit_photo.html

```diff
@@ -40,12 +48,20 @@
                     <h3>Current profile photo</h3>
                 </div>
                 <div class='submodule-body'>
-                    <img src="{{ photo_url }}" />
+                    <img id="profileimage" src="{{ photo_url }}" />
                 </div>
             </div>
             <div class="submodule photo-wide">
+            <div class="notification" id="notification" style="display:none;">
```

Comment:
It looks like the rest of the elements in this file do not use inline styles. Could you move the inline css you added into one of included static css files?